### PR TITLE
Import clang kickstart guide 

### PR DESCRIPTION
**Pull request by axic**
Part of #111.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,8 @@ One of the [design goals](https://github.com/ewasm/design/blob/master/rationale.
 
 At present, we've developed support for the following languages and toolchains:
 
-- LLVM: C, C++, and Rust: documentation pending
+- [C/C++ (LLVM) WebAssembly tutorial](./clang.md)
+- Rust: documentation pending
```
Comment (jakelang) [inline README.md]: There is a PR open for this.

Comment (jakelang) [reply README.md]: #129

Comment (axic) [reply README.md]: I know. Want to merge that too.

**Comment by axic** [comment]
Your PR adds the extra text :)

**Pull request merged**